data/scenarios/Testing/380-counting-objective.yaml

```diff
@@ -0,0 +1,36 @@
+version: 1
+name: Evaluate count of an item
+description: |
+  The goal criteria is exactly the same as
+  379-possession-objective but evaluated
```

Comment:
@kostmo you know that we number the tests based on their issues, right? :raised_eyebrow: 

For example `394-build-drill.yaml` even has a link to `https://github.com/swarm-game/swarm/issues/394`.

These numbers are not issues:
- #380
- #379